### ISSUE TITLE
Trouble scraping deck.tk/deckstats.net

### ISSUE THREAD

**Issue by Triquetra** [howie6879/ruia#135]
For example:
```
import asyncio
from ruia import Request


async def request_example():
    url = "https://deck.tk/07Pw8tfr"
    params = {
        'name': 'ruia',
    }
    headers = {
        'User-Agent': 'Python3.6',
    }
    request = Request(url=url, method='GET', params=params, headers=headers)
    response = await request.fetch()
    json_result = await response.json()
    print(json_result)


if __name__ == '__main__':
    asyncio.get_event_loop().run_until_complete(request_example())
```

This simply hangs without resolution. That is, the request is never resolved, and I must Ctrl-C out of it. Scrapy handles this without issue, but I was hoping to transition to ruia. Any ideas?

**Comment by Triquetra** [comment]
Also:
```
poetry add ruia-shell
ruia_shell "https://deck.tk/07Pw8tfr"
```
never resolves, and must be terminated by Ctrl-C.

**Comment by howie6879** [comment]
> This simply hangs without resolution. That is, the request is never resolved, and I must Ctrl-C out of it. Scrapy handles this without issue, but I was hoping to transition to ruia. Any ideas?

Just add a aiohttp session:

```python
import aiohttp
import asyncio
from ruia import Request


async def request_example():
    url = "https://deck.tk/07Pw8tfr"
    params = {
        'name': 'ruia',
    }
    headers = {
        'User-Agent': 'Python3.6',
    }
    async with aiohttp.ClientSession() as session:
        request = Request(url=url, method='GET', params=params, headers=headers)
        response = await request.fetch()
        json_result = await response.json()
        print(json_result)


if __name__ == '__main__':
    asyncio.get_event_loop().run_until_complete(request_example())
```

**Comment by howie6879** [comment]
> never resolves, and must be terminated by Ctrl-C.

I'll fix this issue soon~

Thanks for your report. :)

**Comment by Triquetra** [comment]
This did not seem to resolve the issue for me. It still hangs without resolving the request.

**Comment by howie6879** [comment]
@Triquetra 

Notice this line: `request = Request(url=url, method='GET', params=params, session = session , headers=headers)`

Sorry, I just forgot to pass in session parameters to request

**Comment by Triquetra** [comment]
Thank you for your help. Including request_session=session as a Request argument solved it.

**Comment by howie6879** [comment]
> never resolves, and must be terminated by Ctrl-C.

Fixed, upgrade ruia-shell to 0.0.4:

```shell
pip install -U ruia-shell
# or
pip install git+https://github.com/python-ruia/ruia-shell.git
